fenix/app/src/test/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckStoreTest.kt

```diff
@@ -1282,6 +1282,55 @@ class ReviewQualityCheckStoreTest {
             assertTrue(observedState.contains(expectedProgressState3))
         }
 
+    @Test
+    fun `GIVEN analysis succeeds WHEN deletedProduct is true and deletedProductReported is false THEN product not available is shown`() =
+        runTest {
+            val tested = ReviewQualityCheckStore(
+                middleware = provideReviewQualityCheckMiddleware(
+                    reviewQualityCheckPreferences = FakeReviewQualityCheckPreferences(isEnabled = true),
+                    reviewQualityCheckService = FakeReviewQualityCheckService(
+                        productAnalysis = {
+                            ProductAnalysisTestData.productAnalysis(
+                                deletedProduct = true,
+                                deletedProductReported = false,
+                            )
+                        },
+                        reanalysis = AnalysisStatusDto.PENDING,
+                        statusProgress = {
+                            AnalysisStatusProgressDto(
+                                status = AnalysisStatusDto.COMPLETED,
+                                progress = 100.0,
+                            )
+                        },
```

Comment:
🔍 These two might not need to be set for this test, seem unrelated

Reply:
Ya this isn't necessary, updated to `{ null }`